apt.txt

```diff
@@ -0,0 +1 @@
+openjdk-8-jre
```

Comment:
Because we're using JDK 8 by default in OSS side :-).
Also, we should set the configurations such as `-Dio.netty.tryReflectionSetAccessible=true` to make it working with Arrow optimization.